SU2_CFD/src/iteration/CFluidIteration.cpp

```diff
@@ -358,7 +375,7 @@ void CFluidIteration::SetWind_GustField(CConfig* config, CGeometry** geometry, C
     NewGridVel[iDim] = 0.0;
   }
 
-  GustDer = new su2double[3];
+  GustDer = new su2double[4];
   for (unsigned short i = 0; i < 3; i++) {
     GustDer[i] = 0.0;
```

Comment:
Watch out that you are not initializing the new entry.

Reply:
Right, thank you for spotting this! Actually, the gust derivatives are part of the Split Velocity Method and unused. So this is part of the code that I believe could/should be removed anyways.